uint32 bill_number
time timestamp
uint32 Quantity
float32 Price
float32 Total
